DataDog/datadog-agent
# [windows] Increase timeout when restarting agent through GUI

### What does this PR do?

Increases timeout from 2s to 10s when sending request to the GUI server to restart the agent, as the current timeout is too aggressive and gets triggered often. With the short timeout, the agent would display "An error occured." when resetting instead of showing the general page again.

### Additional Notes

This is a bit tricky to reproduce. Make sure to restrict CPU on the machine. In my tests, I set a wrong API key on install (I believe this delays restart a bit, causing a timeout to be more likely).

### Describe your test plan

1. Use low CPU machine (I used 1 core)
2. Install agent version 7.22.0 or 7.21.0 
3. Enter fake API Key (makes the error more consistent)
4. Restart agent from GUI, it should display "An error occured." after loading a bit
5. Try the same with this version, it should not display the error.
